styles/pronouns.page/catppuccin.user.css

```
@@ -223,6 +223,10 @@
       color: @text;
       background-color: @mantle;
       border-color: @surface0;
+
+      &:focus {
+        box-shadow: 0 0 0 .25rem fade(@accent-color, 5%)
```

Should be `50%` sorry.
```suggestion
        box-shadow: 0 0 0 .25rem fade(@accent-color, 50%)
```